src/store/ChartStore.js

```diff
@@ -387,6 +397,8 @@ class ChartStore {
         // trading times and active symbols can be reused across multiple charts
         this.tradingTimes = ChartStore.tradingTimes || (ChartStore.tradingTimes = new TradingTimes(this.api, this.mainStore.state.shouldFetchTradingTimes));
         this.activeSymbols = ChartStore.activeSymbols || (ChartStore.activeSymbols = new ActiveSymbols(this.api, this.tradingTimes));
+        this.networkMonitor = NetworkMonitor.getInstance();
```

Comment:
@shayan-binary-2 wouldn't it be part of the app instead of the library while we don't need it when the chart is used as a component in another app.